tasmota/tasmota_xsns_sensor/xsns_44_sps30.ino

```diff
@@ -140,14 +140,17 @@ unsigned char cmdb[6];
 void SPS30_Detect(void)
 {
   if (!I2cSetDevice(SPS30_ADDR)) { return; }
-  I2cSetActiveFound(SPS30_ADDR, "SPS30");
-
   uint8_t dcode[32];
   sps30_get_data(SPS_CMD_GET_SERIAL,dcode,sizeof(dcode));
-  AddLog(LOG_LEVEL_DEBUG, PSTR("sps30 found with serial: %s"),dcode);
+  if(dcode[0] == 0) {
+    AddLog(LOG_LEVEL_DEBUG, PSTR("No serial returned, not an SPS30, maybe a SEN5X?"));
```

Comment:
Pls remove non sense message. Wastes code space and stop advertising your Deense stuff.

Reply:
Sorry, was only an attempt to be helpful not at advertising 🤣